### PR TITLE
SAK-41821 Assignments : student progress bar and confirmation screen

### PR DESCRIPTION
**Pull request by bgarciaentornos**
I'd like to be reviewed not only the code and functionalities, but also the default messages shown and the related properties. Thanks!

### CODE REVIEW

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -1959,6 +1959,20 @@
...
 
+# ######################################
+# SAK-41821 Student confirmation page
+# ######################################
+# Custom text shown for students on the confirmation screen, loaded by default from properties but it can be overriden here
+# assignment.customConfirmMessage=You are about to submit this submission for grading. Please check that the following items have been completed before submitting.
+
+# If set to true, submissionConfirmCheck items will have to be checked to submit
+# DEFAULT: false - submissionConfirmCheck items will appear as a bullet list
+# submissionConfirmChecks=true
```
Comment (ern) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: prefix this option with "assignment."

**Comment by ern** [comment]
It would be ideal to add aria labels where appropriate.

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties (hunk 1 above)
Comment (bgarciaentornos) [reply config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Thanks! Just updated this and also solved some accessibility issues.

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 2]
```diff
@@ -1967,6 +1967,20 @@
 # DEFAULT: 2 (new assignments will use 2 decimal places for grading by default)
 #assignment.grading.decimals=1
 
+# ######################################
+# SAK-41821 Student confirmation page
+# ######################################
+# Custom text shown for students on the confirmation screen, loaded by default from properties but it can be overriden here
+# assignment.customConfirmMessage=You are about to submit this submission for grading. Please check that the following items have been completed before submitting.
```
Comment (ern) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: lets update this property to "assignment.submissionConfirmationMessage"

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 3]
```diff
@@ -1967,6 +1967,20 @@
...
+
+# If set to true, submissionConfirmCheck items will have to be checked to submit
+# DEFAULT: false - submissionConfirmCheck items will appear as a bullet list
+# assignment.submissionConfirmChecks=true
```
Comment (ern) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Can we change this property to "assignment.submissionConfirmationOptions"

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 4]
```diff
@@ -1967,6 +1967,20 @@
...
+
+# assignment.submissionConfirmCheck.count=2
```
Comment (ern) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: and this property to "assignment.submissionConfirmOption"

**Comment by bgarciaentornos** [comment]
@ern I've just renamed the properties.

**Pull request merged**